hwdb.d/60-keyboard.hwdb

```diff
@@ -384,7 +384,7 @@ evdev:name:Dell WMI hotkeys:dmi:bvn*:bvr*:bd*:svnDell*:pnPrecision*:*
 
 # Dell Latitude privacy microphone mute
 evdev:name:Dell Privacy Driver:dmi:bvn*:bvr*:bd*:svnDell*:pnLatitude*:*
- KEYBOARD_KEY_12001=f20                                 # Mic mute toggle, should be micmute
+ KEYBOARD_KEY_120001=f20                                 # Mic mute toggle, should be micmute
```

Comment:
doesn't matter, but maybe remove one space char now, to keep the # aligned with the lines further up

Reply:
> doesn't matter, but maybe remove one space char now, to keep the # aligned with the lines further up

fixed just now .
thanks